src/components/itemcontextmenu.js

```diff
@@ -305,6 +310,22 @@ define(['apphost', 'globalize', 'connectionManager', 'itemHelper', 'appRouter',
                         getResolveFunction(getResolveFunction(resolve, id), id)();
                     });
                     break;
+                case 'copy-stream':
+                    var downloadHref = apiClient.getItemDownloadUrl(itemId);
+                    var textArea = document.createElement("textarea");
+                    textArea.value = downloadHref;
+                    document.body.appendChild(textArea);
+                    textArea.focus();
+                    textArea.select();
+                    try {
+                        document.execCommand('copy');
+                    } catch (err) {
+
```

Comment:
Shouldn't we at least write the error to the console?

Reply:
Fixed. Thanks for the review.